charts/rancher-monitoring/v0.1.3/values.yaml

```diff
@@ -314,16 +314,16 @@ prometheus:
   apiGroup: "monitoring.coreos.com"
   image:
     repository: rancher/prom-prometheus
-    tag: v2.17.2
+    tag: v2.21.0
```

Comment:
re: https://github.com/rancher/image-mirror/pull/35#discussion_r489704012, this should be `v2.18.2` since `kube-prometheus-stack` (the new upstream) has not bumped to this version yet.